src/domUpdates.js

```diff
@@ -109,9 +111,31 @@ class DomUpdates {
   greetUser(currentUser) {
     const userName = document.querySelector('.user-name');
     userName.innerHTML = `
-    Welcome ${currentUser.name.split(' ')[0]} ${currentUser.name.split(' ')[1][0]}.`;
+    Welcome ${currentUser.name.split(' ')[0]} ${currentUser.name.split(' ')[1][0]}.
+    `
   }
+//should update the dom
+  //after the user has clicked on the images the message should change
+  displayIngredientFeedback(feedback, id) {
+    console.log(id)
+    let card = document.getElementById(id)
+    let name = document.getElementById(`${id} name`)
+    card.innerHTML = ``;
+    card.insertAdjacentHTML('beforeend', `
+    <button class="back-button">No Thanks</button>
+    <button id="${id}" class="purchase-button">Add Ingredients</button>
+    `)
+    feedback.forEach(ingredient => card.innerHTML += `<ul><li>${ingredient}</li></ul>`)
+  }
+
+  updatePurchase(messages, id){
+    let card = document.getElementById(id)
+    const values = messages.map( message => message.message)
+    card.innerHTML = ``;
+    values.forEach(message => card.innerHTML +=`<ul><li>${message}</li></ul>`)
+    card.insertAdjacentHTML('beforeend', `
+    <i class="fas fa-arrow-circle-left back-button"></i>`)
+ }
```

Comment:
Looks good. I am assuming this is where you retrieve the data from get POST fetch, yeah? Looks like you got it working, good job!

Reply:
the posting actually happens in the `postIngredients` on scripts.js that is triggered when the user clicks on the `purchase-button`, which now that I am thinking about we should probably use a better naming conventions

Reply:
is the user making a new purchase or just adding to the pantry, I feel it should just be added to the pantry since there is no functionality we have as of right now that is checking for a `wallet` or some way to track the user expenses, let me know that you think.

Reply:
Yeah that makes sense, I more meant that this is sort of the "end of the line" with our POST data flow, this is the function that will display the updated pantry information for our user.

Reply:
I agree we should just word this add "add to pantry" or something along those lines. Any sort of mock payment system is a little out our scope on this one :)